README.md

```diff
@@ -2,3 +2,7 @@
 Some programs doing the basic functions in Tweepy (Twitter for Python).
 
 A sample python.json file has been included.
+
+# Required Dependency
```

Comment:
A `requirements.txt` file with `pip install -r requirements.txt` would also be great in addition to this, do you agree?

Reply:
Yeah. I agree with you. I was thinking about that earlier